### PR TITLE
[Benchmark] fix file path

### PR DESCRIPTION
**Pull request by Dania-Abuhijleh**
fix file path in file_exists function

### CODE REVIEW

File: benchmark-service/deployment.yaml [hunk 1]
```diff
@@ -37,13 +37,11 @@ spec:
 {% if deploy %}
           - name: HAIL_BENCHMARK_BUCKET_NAME
             value: hail-test
+          - name: START_POINT
```
Comment (jigold) [inline benchmark-service/deployment.yaml:40]: I don't understand why this change would fix the behavior we were seeing. Can you explain?

File: benchmark-service/benchmark/utils.py [hunk 2]
```diff
@@ -88,6 +88,7 @@ def file_exists(self, file_path):
         file_info = parse_file_path(FILE_PATH_REGEX, file_path)
         bucket_name = file_info['bucket']
         bucket = self.storage_client.bucket(bucket_name)
-        exists = storage.Blob(bucket=bucket, name=file_path).exists()
-        log.info(f'file in bucket {bucket_name} exists? {exists}')
+        shorter_file_path = file_info['path']
```
Comment (jigold) [inline benchmark-service/benchmark/utils.py:91]: This is just `path`. There's two components: the bucket and the path.

**Review by jigold — CHANGES_REQUESTED** [review]

File: benchmark-service/deployment.yaml (hunk 1 above)
Comment (Dania-Abuhijleh) [reply benchmark-service/deployment.yaml:40]: It wouldn't, I just thought it was repetitive to have {% if deploy %} twice

File: benchmark-service/benchmark/utils.py (hunk 2 above)
Comment (Dania-Abuhijleh) [reply benchmark-service/benchmark/utils.py:91]: I thought I got the bucket on line 89, is there something else?

Comment (jigold) [reply benchmark-service/benchmark/utils.py:91]: You did. I'm saying I don't like the name `shorter_file_path`. Just name it `path`

File: benchmark-service/deployment.yaml (hunk 1 above)
Comment (jigold) [reply benchmark-service/deployment.yaml:40]: Ok. There's still a problem with START_POINT. The deployment logs says this is the START_POINT:

```
2020-11-03 22:03:59,472 start point is 2020-11-02T21:38:53Z
```

File: benchmark-service/benchmark/utils.py (hunk 2 above)
Comment (Dania-Abuhijleh) [reply benchmark-service/benchmark/utils.py:91]: ahhh

**Review by jigold — APPROVED** [review]
